packages/amplify-codegen/src/utils/validateDartSDK.js

```diff
@@ -10,13 +10,13 @@ function validateDartSDK(context, projectRoot) {
     const config = yaml.load(fs.readFileSync(path.join(projectRoot, PUBSPEC_FILE_NAME), 'utf8'));
     const version = semver.minVersion(config.environment.sdk);
     if (semver.satisfies(version, '>= 2.12.0')) {
-      context.print.warning('\nDetected Dart SDK version: “>= 2.12.0” is “true”');
+      context.print.warning('\nDetected Dart SDK version 2.12.0 or above.');
       return true;
     }
-    context.print.warning('\nDetected Dart SDK version: “>= 2.12.0” is “false”.');
+    context.print.warning('\nDetected Dart SDK version below 2.12.0.');
     return false;
   } catch (e) {
-    context.print.warning('\nCould not detect Dart SDK version, defaulting to “>= 2.12.0”.');
+    context.print.warning('\nCould not detect Dart SDK version, defaulting to 2.12.0.');
```

Comment:
Should we remove the full stop at the end of these statements? It looks like another decimal point.
Also we don't seem to have them in the other warnings.

Reply:
Makes sense to me - I updated the suggestion.